components/datetime/src/options/preferences.rs

```diff
@@ -54,6 +59,25 @@ impl Bag {
             hour_cycle: Some(h),
         }
     }
+
+    /// Construct a [`Bag`] from a given [`DataLocale`]
+    pub fn from_data_locale(data_locale: &DataLocale) -> Self {
```

Comment:
nit: From impl?

Reply:
maybe, but I want to make this hidden behind the "experimental" flag (will do that post-beta, pre-final) and I find the explicit function more clear right now